assets/dev/scss/frontend/_container.scss

```diff
@@ -148,19 +166,34 @@
 	// TODO: Find a better solution.
 	& > & {
 		> .elementor-element-overlay {
-			// Hide any setting control which isn't the edit control.
 			.elementor-editor-element-setting {
-				&:not( .elementor-editor-element-edit ),
-				&:first-child::before {
+				position: relative;
+				background-color: $editor-darker;
+
+				&:hover {
+					background-color: $editor-darkest;
+				}
+
+				// Hide any unwanted setting controls.
+				&.elementor-editor-element-add,
+				&.elementor-editor-element-edit + .elementor-editor-element-remove,
```

Comment:
Try to use or add one class name for all of them (including the “Duplicate” when “Editing Handles” is on)

Reply:
Done (did the best I can)